101-asp-net-app-service/GettingStarted.md

```diff
@@ -0,0 +1,159 @@
+|workerSizeId   |Gets or sets size ID of machines: 0 - Small 1 - Medium 2 - Large   |
+|numberOfWorkers   |Gets or sets number of workers.   |
+|hostingPlanName   |Name for the hosting plan. On free tier, you can only have 1 linux hosting environment.   |
+|currentStack   |Framework used on the hosting   |
+|netFrameworkVersion   |Framework version   |
+
+
+#### Resources
+
+The resources are the services that we will to deploy into Azure. In this template we will deploy three resources:
+
+|**RESOURCE NAME**   |**DESCRIPTION**   |
+|---|---|
+|Microsoft.Web/sites   |This is our App Service.   |
+|Microsoft.Web/serverfarms   |This is the hosting for our App Service.  |
+|microsoft.insights/components   |Used to monitor our live web application   |
+
+## Deployment
+
+There are a few ways to deploy this template.
+You can use [PowerShell](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-template-deploy), [Azure CLI](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-template-deploy-cli), [Azure Portal](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-template-deploy-portal) or your favorite SDK.
+
+For this task, we will deploy using the Portal and Azure CLI, I choose to use the Visual Code with Azure CLI extensions, if you like, you can find more information [here](https://code.visualstudio.com/docs/azure/extensions). But bare in mind that you don't need to use the Visual Code, you can stick with the old good always present **Command Line** on Windows or any **bash terminal**.
+
+### Using Azure CLI with Visual Code
+Type on the terminal windows: 
+
+```
+az login
+```
+
+![Screen](./images/az-log.png)
+
+
+You gonna be redirected to the Azure Portal where you can use your credentials to login into.
+
+After login, you gonna have your credentials. 
+
+In order to set the right subscription, you can use the follow command:
+
+```
+az account set --subscription "< your subscription id >"
+```
+
+### Resource Group
+
+A resource group is a container that holds related resources for an Azure solution. The resource group includes those resources that you want to manage as a group. 
+
+We gonna need to create a resource group for our deployment if we haven't yet create a resource group.
+
+To create a resource group, we will need a name and the location. For a list of locations, type:
+
+```
+az account list-locations
+```
+
+To create the resource group, just type the command:
+
+```
+az group create --name <mygroupname> --location <thelocation>
+```
+
+![Screen](./images/az-groupcreate.png)
+
+Now, we are ready to deploy our template. Type the next command to get it:
+
+```
+az group deployment create --resource-group <your resource-group name> --template-file <full path and name of your template>
+```
+
+![Screen](./images/az-group-deploy.png)
+
+It will spend some time on the deployment. Bring it like 5 minutes until the process finish.

```

Comment:
fix grammar - there are quite a few more grammatical errors in this file, I won't comment on them all so should have someone do a quick review